chia/full_node/full_node.py

```diff
@@ -97,6 +97,7 @@ class FullNode:
     _blockchain_lock_ultra_priority: LockClient
     _blockchain_lock_high_priority: LockClient
     _blockchain_lock_low_priority: LockClient
+    _transaction_queue_task: Optional[asyncio.Task]
```

Comment:
this could be initialized to `None` here, right?

Reply:
Thanks for taking a look here.  `FullNode` isn't defined using `@dataclass` so assignment here would be creating a class attribute (global) not an instance attribute default.  I can explain and share an example if you aren't already familiar with the implications.